controller/app.controller.js

```diff
@@ -20,13 +20,40 @@ const wrongPath = (req, res, next) => {
 };
 
 const getArticleById = (req, res, next) => {
-    selectArticle(req.params.id)
+    selectArticleById(req.params.id)
         .then((data) => {
-            res.status(200).send({ rows: data });
+            if (data.length === 0){
```

Comment:
Ideally this error handling would be in your Model, rather than controller.
Handling errors related to data storage, retrieval, and validation in the model layer helps maintain a clean separation of concerns.